frontend/packages/data-portal/app/components/Deposition/DepositionMetadataDrawer/DepositionMetadataDrawer.tsx

```diff
@@ -47,7 +49,7 @@ function MetadataTab() {
   )
 }
 
-function HowToCiteTab() {
+function MethodSummaryTab() {
```

Comment:
This and the function above should be moved to their own files to maintain 1 component definition per file.